eng/pipelines/common/xplat-setup.yml

```diff
@@ -107,12 +107,12 @@ jobs:
         # Public Linux Build Pool
         ${{ if and(in(parameters.osGroup, 'Linux', 'FreeBSD', 'Browser', 'Android'), eq(variables['System.TeamProject'], 'public')) }}:
           name:  NetCorePublic-Pool
```

Comment:
- You can't use different queue names here, you'll need to merge your changes
- Need to change all instances of "NetCorePublic-Pool" to "NetCorePublic-Int-Pool"